containers/helm/values.yaml

```diff
@@ -3,25 +3,55 @@ nameOverride: ""
 fullnameOverride: ""
 replicaCount: 1
 
-deployment:
-  labels:
-    app: "pwpush"
-    # component: ingress-controller
-    # type: nginx
-  containerPort: 5100
-  readinessProbe:
-    enabled: true
-    path: /
-  livenessProbe:
-    enabled: true
-    path: /
-
 image:
   repository: docker.io/pglombardo/pwpush
   pullPolicy: Always
   tag: "release"
 
+labels:
+  app: "pwpush"
+podLabels:
+  app: "pwpush"
+
+env: {}
```

Comment:
I like the approach from Mend Renovate CE that they put application login under its own section.  These envs will explicitly be for pwpush?

https://github.com/mend/renovate-ce-ee/blob/5432fc8dbfb09390763fbade48afb198c060753c/helm-charts/mend-renovate-ce/values.yaml#L10